docs/source/changelog/0.6.0-changelog.rst

```diff
@@ -0,0 +1,474 @@
+
+   ``NumberLineOld`` was introduced to continue support for :class:`~.GraphScene`, although we are moving away from GraphScene and intend to deprecate it in a future release.
+
+* `#1013 <https://github.com/ManimCommunity/manim/pull/1013>`__: Refactored the Command Line Interface to use Click instead of Argparse
+   This change breaks the CLI API to organize the structure of Manim Community's commands, options, and arguments.
+
+   To be more in line with POSIX compliant CLI conventions, options for commands are given *BEFORE* their arguments.
+   In Argparse: ``manim basic.py -p -ql``
+   With Click: ``manim -p -ql basic.py``
+
+   Although this is primarily a refactor and most of the common options are still there, some options have been added/removed. Use the ``manim`` command's ``-h``, ``--help`` option, or simply run the command without providing options/arguments to view the help page with the full list of subcommands/options/arguments.
+
+   - Added a ``--fps``/``--frame_rate`` option which allows for custom fps that don't have to be integer (i.e. 29.97, 23.98, etc.). Users no longer have to specify the FPS from within a config file. Additionally, the ``--webgl_renderer_fps`` option has been removed. Use ``--fps`` or ``--frame_rate`` instead.
+   - Added a ``--renderer`` option which you can use to select your choice of renderer (e.g. ``--renderer=opengl``). There are currently *THREE* renderers to choose from!
+   - Removed the ``--background_color`` option. Reassigned the ``--background_color`` option's shorthand ``-c`` to ``--config_file``.
+   - Removed the ``--leave_progress_bars`` option. Use ``--progress_bars=leave`` instead.
+   - Removed the deprecated render quality flags, in particular: ``-l``, ``-m``, ``-h``, ``-k``.
+   - Removed the ``--sound`` option. It lost support long ago with the removal of SoX.
+
+Deprecated classes and functions
+--------------------------------
+
+* `#1354 <https://github.com/ManimCommunity/manim/pull/1354>`__: Refactored a few functions in space_ops.py, deprecated :func:`~.angle_between`
+
+
+* `#1370 <https://github.com/ManimCommunity/manim/pull/1370>`__: Remove TexMobject and TextMobject
+   TexMobject and TextMobject have been deprecated for a while, they are now fully removed. Use Tex or MathTex instead.
+
+* `#1349 <https://github.com/ManimCommunity/manim/pull/1349>`__: Removed the deprecated ``SmallDot`` mobject
+
+
+* `#1259 <https://github.com/ManimCommunity/manim/pull/1259>`__: Removed deprecated CairoText class
+   - Removed `CairoText` class from `manim.mobject.text_mobject`
+    - Removed `'CairoText'` from `__all__`
+    - Removed reference to `CairoText` in docstring
```

Comment:
This indentation needs to be fixed, but I think this s3ection is honestly pretty redundant:
```suggestion

```